src/java/org/apache/cassandra/exceptions/RequestFailureReason.java

```diff
@@ -47,11 +50,9 @@ public enum RequestFailureReason
     NOT_CMS                                 (8),
     INVALID_ROUTING                         (9),
     COORDINATOR_BEHIND                      (10),
+    RETRY_ON_DIFFERENT_TRANSACTION_SYSTEM   (11),
     // The following codes have been ported from an external fork, where they were offset explicitly to avoid conflicts.
     INDEX_BUILD_IN_PROGRESS                 (503),
-    RETRY_ON_DIFFERENT_TRANSACTION_SYSTEM   (504),
-    BOOTING                                 (505),
-    ACCORD_DISABLED                         (506)
```

Comment:
i migrated to 5xx in the rebase but confirmed with caleb that the 503 thing is specific to `INDEX_BUILD_IN_PROGRESS` and shouldn't be maintained.  I didn't go back and update the history because this file conflicted many times, so had to fix it ~6 times or so... if desired i can try to fix the history rather than put it here

Reply:
I have questions as to why the Apache C* should use indexing based on a external fork, but yeah this looks correct.